src/OSPSuite.SimModelNative/src/TableFormulaWithOffset.cpp

```diff
@@ -155,7 +155,9 @@ void TableFormulaWithOffset::Finalize()
 
 vector <double> TableFormulaWithOffset::RestartTimePoints()
 {
-	assert ((_tableObject != NULL) && (_offsetObject != NULL));
+	//table formula can be NULL if the original table formula was constant and thus was replaced by its value
+	if (_tableFormula == NULL)
+		return vector<double>();
```

Comment:
in the original example, the table formulation was constantly zero; the table formula was replaced early in the process with its const value (zero) and the old line 172 below crashed then